src/pactWith.ts

```diff
@@ -11,7 +11,11 @@ const setupProvider = (options: JestPactOptions): Pact => {
 
   beforeAll(() => pactMock.setup());
   afterAll(() => pactMock.finalize());
-  afterEach(() => pactMock.verify());
+  // Fails if enabled - Error in native callback
+  //   Error in native callback
+  //   at Object.mockServerMismatches (node_modules/@pact-foundation/pact-core/src/consumer/index.ts:94:13)
+  //   at Object.<anonymous> (src/index.ts:37:28)
+  // afterEach(() => pactMock.verify());
```

Comment:
test failures if uncommented

https://github.com/pact-foundation/pact-js/blob/master/MIGRATION.md#beta61

> the mockService property on the Pact class is no longer an actual MockService, but supports the baseUrl property.

Reply:
See https://github.com/pact-foundation/jest-pact/pull/214#issuecomment-1205206648 for why this was failing